test/test_normal_form_game.jl

```diff
@@ -260,6 +284,59 @@
                 end
             end
         end
+
+        @testset "Test is_dominated" begin
+            coordination_game_matrix = [4 0; 3 2]
+            player = Player(coordination_game_matrix)
+            for action = 1:num_actions(player)
+                @test !is_dominated(player, action)
+            end
+
+            payoffs_2opponents = Array{Int64}(undef, 2, 2, 2)
+            payoffs_2opponents[:, 1, 1] = [3, 1]
+            payoffs_2opponents[:, 1, 2] = [6, 0]
+            payoffs_2opponents[:, 2, 1] = [4, 5]
+            payoffs_2opponents[:, 2, 2] = [2, 7]
+            player = Player(payoffs_2opponents)
+
+            for i = 1:num_actions(player)
+                @test !is_dominated(player, i)
+            end
+
+        end
+
+        @testset "Test player corner cases" begin
+            n, m = 3, 4
+            player = Player(zeros((n, m)))
+            for action = 1:n
+                @test is_best_response(player, action, ones(m) * 1/m)
+                @test !is_dominated(player, action)
+            end
+
+            e = 1e-8
+            player = Player([-e -e;
+                             1 -1;
+                             -1 1])
+            action = 1
+            @test is_best_response(player, action, [1/2, 1/2], tol=e)
+            @test !is_best_response(player, action, [1/2, 1/2], tol=e/2)
+            @test !is_dominated(player, action, tol=e+1e-16)
+            @test dominated_actions(player, tol=e+1e-16) ==
+                  Vector{Integer}(undef, 0)
+            @test is_dominated(player, action, tol=e/2)
+            @test dominated_actions(player, tol=e/2) == [action]
+        end
+
+        @testset "Test rational input game" begin
+            game_matrix = [2//3 1//3;
+                           1//3 2//3]
```

Comment:
Please test with a "corner case". You may use the above case with, e.g., `e` = `0//1` and `1//(10^8)`.